code/modules/paperwork/paperplane.dm

```diff
@@ -104,7 +104,7 @@
 		H.emote("scream")
 
 /obj/item/weapon/paper/AltClick(mob/living/carbon/user, obj/item/I)
-	if((!in_range(src, user)) || usr.stat || usr.restrained())
+	if((!in_range(src, user)) || usr.stat || usr.restrained() || (issilicon(user)) )
```

Comment:
ohhhhh that makes sense, I thought he wanted me to `!istype(user, silicon path)`